src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -19,16 +22,26 @@ public class DataAnnotationsClientModelValidatorProvider : IClientModelValidator
     {
         // A factory for validators based on ValidationAttribute.
         internal delegate IClientModelValidator
-            DataAnnotationsClientModelValidationFactory(ValidationAttribute attribute);
+            DataAnnotationsClientModelValidationFactory(ValidationAttribute attribute, IStringLocalizer stringLocalizer);
```

Comment:
this would be much easier to read if you put the line break after `DataAnnotationsClientModelValidationFactory(`